Competition2017/src/competition/subsystems/autonomous/ShootAndDriveAcrossBaseLineCommandGroup.java

```diff
@@ -49,13 +52,16 @@ public ShootAndDriveAcrossBaseLineCommandGroup(XPropertyManager propManager,
         driveBackABit = driveForDistanceProvider.get();
         driveBackABit.setDeltaDistance(distanceToBackUp);
         
+        this.addSequential(driveBackABit, 0.25);
+        this.addParallel(stopFiring, 0.25);
+        
         //aim away from driver station (towards baseline)
         rotateToBaseline.setTargetHeading(90);
-        this.addSequential(rotateToBaseline);
```

Comment:
Our rotate-to-heading was tuned for low gear, but the robot was doing autonomous in high gear.
Either the robot needs to shift during these auto programs, or we need different tuning.